lib/constants.ts

```diff
@@ -141,7 +141,8 @@ export const supportedNotificationEvents = new Set([
     's3:ObjectRestore:Post',
     's3:ObjectRestore:Completed',
     's3:ObjectRestore:Delete',
-    's3:LifecycleTransition'
+    's3:LifecycleTransition',
+    's3:LifecycleExpiration'
```

Comment:
should we also have the other "sub" events ?

`s3:LifecycleExpiration:Delete`
`s3:LifecycleExpiration:DeleteMarkerCreated`